internal/models/quizzes/quizzes.go

```diff
@@ -206,7 +206,7 @@ func scanQuizFromFullRow(row *sql.Row) (*Quiz, error) {
 	}
 	quiz.ImageURL = *tempURL
 
-	if err == sql.ErrNoRows {
+	if err != nil {
```

Comment:
from what i see here  the err may come from `data_handling.ConvertNullStringToURL...`
i do believe and agree that the `sql.ErrNoRows` was out of place here, but isn't the entire second `if err` pointless? this error is already checked for on line 204. There is no new error being returned at any point after that check